scss/styles.scss

```diff
@@ -1,3 +1,20 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
```

Comment:
You can create a `titles.scss` also

Reply:
I will remove it. And I will add it in the page-title PR